WTWSMS/decisions/WtWSMS_religious_decisions.txt

```diff
@@ -407,57 +407,72 @@ decisions = {
 	create_priscillian_patriarchy = {
 		only_playable = yes
 		is_high_prio = yes
-	
 		potential = {
 			religion = priscillianism
 			NOT = { is_title_active = d_priscillianism }
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
 		}
-		
 		allow = {
-			is_heretic = no
 			piety = 1000
+			trait = vegetarian
+			martial = 8
+			learning = 8
```

Comment:
These conditions are inconsistent: we don't use learning or martial rating as conditions.